build.cake

```diff
@@ -29,7 +29,11 @@ var ANDROID_SDK_BASE_VERSION = "v1.0";
 var ANDROID_SDK_VERSION = "v9.0";
 if (string.IsNullOrEmpty(XAMARIN_ANDROID_PATH)) {
 	if (IsRunningOnWindows()) {
-		var vsInstallPath = VSWhereLatest(new VSWhereLatestSettings { Requires = "Component.Xamarin" });
+		var vsInstallPath = VSWhereLatest(new VSWhereLatestSettings 
+		{ 
+			Requires = "Component.Xamarin",
+			IncludePrerelease = true

```

Comment:
This should probably be specified from a cake argument and default to false? Is the default behaviour of vswhere to return stable first?